seisbench/models/base.py

```diff
@@ -586,6 +579,38 @@ def _contains_callable_recursive(dict_obj):
 
         seisbench.logger.debug(f"Saved {self.name} model at {path.absolute}")
```

Comment:
This raises an error on string paths currently.

```python
AttributeError: 'str' object has no attribute 'absolute'
```

 I know we usually default to ```Path``` objects for everything but for ease-of-use during model saving and loading I would allow for this to be done using strings too which are then converted to Path objs. So just have the DEBUG line log the absolute path for bath cases accounting for whether the path parameter was a string or path object.

Reply:
Fixed that. Interesting way to introduce errors by factoring out a function. Great catch!